core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -1076,6 +1076,32 @@ public static int levenshtein(String string1, String string2) {
     return LEVENSHTEIN_DISTANCE.apply(string1, string2);
   }
 
+  /** SQL FIND_IN_SET(string, stringArray) function. */
+  public static int findInSet(String matchStr, String textStr) {
+    if (matchStr.contains(",")) {
+      return 0;
+    }
+    int textStrLen = textStr.length();
```

Comment:
final int textStrLen = textStr.length();

Reply:
Thanks for catching this, I will improve that.

Reply:
Resolved.